PhilipFunkos/src/home.css

```diff
@@ -0,0 +1,14 @@
+.home-header {
+  font-size: 50px;
+}
+.home-about-us {
```

Comment:
You can connect multiple selectors together if they share same styles, like this:

```
.home-about-us,
.home-hours {
  /// ...
 }
 ```